### DISCUSSION TITLE
Default Album Art Not Changing

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4568] [Q&A] [answered]
Hi, in Custom Branding, I entered a new url for default album art, but the old image keeps displaying in file uploads. I have re-started the station and done docker down/up.

Does anyone have any other suggestions to get new album art to display?

**Comment by countrymusicfy** [comment, marked as answer]
Have to change default album art in station's edit profile, not custom branding

**Comment by BusterNeece** [comment]
If anyone's curious about this in the future, basically the following order applies when looking for a default album art to use:

 - If the specific station has a custom album art URL,
 - If the system-wide settings have a custom album art URL, then
 - If there is a custom art file uploaded in the custom branding.
